app/controllers/documents_controller.rb

```diff
@@ -7,7 +7,16 @@ class DocumentsController < ApplicationController
   end
 
   def index
-    @documents = Document.page(params[:page]).per(50)
+    filter = DocumentFilter.new(
+      filters: params.permit(:title_or_url, :document_type).to_hash,
+      sort: params[:sort],
+      page: params[:page],
+      per_page: 50,
+    )
+
+    @documents = filter.documents
```

Comment:
Perhaps consider [passing in a single instance variable](https://robots.thoughtbot.com/sandi-metz-rules-for-developers) here.